### PR TITLE
PowerShell Core 6.0 release notes

### PR DESCRIPTION
**Pull request by joeyaiello**
Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [x] This issue only shows up in version 6.0 of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

I just made it through the CHANGELOG, still rife with TODOs, but I wanted to get a general sense from @sdwheeler if this is the right level of abstraction that he was expecting. I'd still like to do a little more thematic, summary language on some of these groupings of copy/pasted changelog items (like in the web cmdlets, for instance), so don't review with *too* much of a fine tooth comb just yet. :)

### CODE REVIEW

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md)

<a name="reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md"></a>
### [reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md)
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801081717423400-2020%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zCa6lfafx5X7POadYj%2FUwk%2Bq%2BK5bVMkc0XX%2Bi4qncBY%3D&st=2018-01-08T17%3A47%3A08Z&se=2018-02-08T17%3A52%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
+# What's New in PowerShell Core 6.0
+
+## Renamed `powershell(.exe)` to `pwsh(.exe)`
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md]: Maybe change heading to be more about Windows PowerShell and PowerShell Core 6 side-by-side and this decision was made, in part, to support this?

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+- Remove parameters `-importsystemmodules` and `-psconsoleFile` from `pwsh.exe`. (#4995)
+- Fix `pwsh -version` and built-in help for `pwsh.exe` to align with other native tools. (#4958 & #4931) (Thanks @iSazonov)
+- Make invalid argument error messages for `-File` and `-Command` consistent and make exit codes consistent with Unix standards (#4573)
+- Add `-WindowStyle` parameter on Windows. (#4573)
+
+## Moved from .NET Framework to .NET Core
+
+TODO
+
+## Support for macOS and Linux
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md]: This should be much higher up.  Probably right after `Windows PowerShell and PowerShell Core 6 side-by-side`

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+- Add support for native command globbing on Unix platforms.
+- `more` function respects the Linux `$PAGER`; defaults to `less`
+ This means you can now use wildcards with native binaries/commands (e.g. `ls *.txt`). (#3463)
+- Escape trailing backslash when dealing with native command arguments. (#4965)
+- Ignore the `-ExecutionPolicy` switch when running PowerShell on non-Windows platforms because script signing is not currently supported. (#3481)
+- Fix ConsoleHost to honour `NoEcho` on Unix platforms. (#3801)
+- Fix `Get-Help` to support case insensitive pattern matching on Unix platforms. (#3852)
+- `powershell` man-page added to package
+
+### Logging
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md]: Might be worth mentioning that on Windows, we have our own PowerShell channel which is distinct from Windows PowerShell

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+
+On macOS, PowerShell uses the native `os_log` APIs to log to Apple's [unified logging system][os_log].
+On Linux, PowerShell uses [Syslog][], a ubiquitous logging solution.
+
+### Filesystem
+
+A number of changes have been made on macOS and Linux to support filename characters not traditionally supported on Windows:
+
+- Paths given to cmdlets are now slash-agnostic (both / and \ work as directory separator)
+- XDG Base Directory Specification is now respected and used by default.
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md]: Not sure if this is the canonical source, but should link to the spec https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+ and the user module path is located at `~/.local/share/powershell/Modules`.
+- Enable support of folders and files with colon in name on Unix. (#4959)
+- Script names or full paths can have commas. (#4136) (Thanks to @TimCurwick!)
+- Fix detection of whether `-LiteralPath` was used to suppress wildcard expansion for navigation cmdlets. (#5038)
+- Update `Get-ChildItem` to be more in line with the way that the *nix `ls -R` and the Windows `DIR /S` native commands handle symbolic links to directories during a recursive search.
+ Now, `Get-ChildItem` returns the symbolic links it encountered during the search, but it won't search the directories those links target. (#3780)
+
+### Case sensitivity
+
+Generally speaking, PowerShell tries to be case insensitive, while macOS and Linux tend to be case sensitive
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md]: This isn't correct.  macOS filesystem seems to be case insensitive by default but can be case sensitive.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md [hunk 6]
```diff
@@ -0,0 +1,345 @@
...
+In some cases, functionality was removed or is unavailable, usually due to a missing dependency in underlying .NET layers,
+but we've done our best to limit these cases.
+
+Most of the modules that ship as part of Windows (e.g. `DnsClient`, `Hyper-V`, `NetTCPIP`, `Storage`, etc.) and other Microsoft products including Azure and Office have not been *explicitly* ported to .NET Core yet.
+However, we are working with these product groups and teams to validate and port their existing modules on PowerShell Core.
+with the promise of .NET Standard and [CDXML](https://msdn.microsoft.com/en-us/library/jj542525(v=vs.85).aspx), many of these traditional Windows PowerShell modules work great in PowerShell Core.
+
+TODO
+
+## Docker support
```
Comment (SteveL-MSFT) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md]: Should this be container support instead of Docker?

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md)

<a name="reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md"></a>
### [reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md)
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 50**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 123**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 301**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 325**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 336**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801100036107557-2020%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vRv%2FiTjEUveBgjkkJmj4QThjMKyoKI1luNT35rvOXwg%3D&st=2018-01-10T00%3A59%3A42Z&se=2018-02-10T01%3A04%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md)

<a name="reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md"></a>
### [reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md)
 - **Line 136**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 136**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 136**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 136**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 136**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 315**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 315**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 315**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 315**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 315**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 335**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 335**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 335**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 335**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 335**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 345**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 345**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 345**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 345**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).
 - **Line 345**: **[Warning]** Invalid file link:(~/docs-conceptual/whats-new/TODO).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801100149041860-2020%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0UVmhqQgcS9SqkH9eHHnaNP3n60I7LeXxbA385gLCDk%3D&st=2018-01-10T02%3A14%3A40Z&se=2018-02-10T02%3A19%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md)

<a name="reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md"></a>
### [reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md)
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801100900202077-2020%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=I%2BtRj13kP3EjVL39oBci%2BAW7YEEwxASCo4X9%2Fpe6Mdo%3D&st=2018-01-10T09%3A22%3A07Z&se=2018-02-10T09%3A27%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by joeyaiello** [comment]
Those links are dependent on another PR, but they will work. Ready to merge this one.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md)

<a name="reference%2Fdocs-conceptual%2Fwhats-new%2FWhat-s-New-in-PowerShell-Core-60.md"></a>
### [reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md](https://github.com/joeyaiello/PowerShell-Docs/blob/ReleaseNotes/reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md)
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).
 - **Line 173**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/SSH-Remoting-in-PowerShell-Core.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801101928166036-2020%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=76d9fuIvr09lu9RGqiyfMqE4VBHtZ%2FyRcUbfBqP%2Fn7E%3D&st=2018-01-10T19%3A52%3A01Z&se=2018-02-10T19%3A57%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**